### PR TITLE
New, more distinguished icons for fullscreen component

### PR DESCRIPTION
**Pull request by scania-bootstrap**
New icons required changes in both the navigation components as well as in the fullscreen component.

### CODE REVIEW

**Comment by scania-bootstrap** [comment]
Here's the initial issue #420

File: src/components/fullscreen/style.less [hunk 1]
```diff
@@ -48,9 +72,32 @@
...
+      }
+    }
+  }
+}
+
+:host-context(.app.header-is-sticky) {
+
+  a {
+
+    .icon-fullscreen {
```
Comment (awinny) [inline src/components/fullscreen/style.less]: This rule will be applied to all fullscreen icon, anywhere you place it. Is this really intended?

Comment (scania-bootstrap) [reply src/components/fullscreen/style.less]: Noop. Will look into it.

**Comment by awinny** [comment]
Will close this for now

**Pull request closed without merging**